charms/unit_test.py

```diff
@@ -200,10 +200,21 @@ def patch_module(fullname, replacement=None):
     """
     Patch a module (and potentially all of its parent packages).
     """
+    patched = []
+
+    def _unpatch(*a):
```

Comment:
This should probably be `_` instead of `a` if it's not needed, since coder linters understand that and will ignore it.